### DISCUSSION TITLE
Error handling

### DISCUSSION THREAD

**Discussion by sseppi** [noi-techpark/it.bz.opendatahub.databrowser#161] [Ideas]
In this discussion will be collected feedback and/or new ideas about the error handling (e.g. positioning, errors messages to be managed, etc.)

**Comment by mrabans** [comment]
In order to show messages, warnings and errors we need a place to put such information. This could be the toolbox with a dedicated tab, called "message box".

Use cases would be:

- Error messages from the API i.e. validation errors, etc
- Information on endpoints, i.e. "this endpoint is deprecated"
- Additional information on realtime data i.e. latency, etc.
